### PR TITLE
[12.0][ADD] product_operating_unit : New module to introduce the operating unit to product template

### PR DESCRIPTION
**Pull request by BT-rmartin**

### CODE REVIEW

File: product_operating_unit/models/product_template.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    def _default_operating_unit_ids(self):
+        return [(6, 0,
+                 [self.env['res.users'].operating_unit_default_get(
+                     self.env.uid).id])]
+
+    operating_unit_ids = fields.Many2many(
+        'operating.unit', 'product_operating_unit_rel',
+        string='Operating Unit',
+        default=_default_operating_unit_ids
+    )
```
Comment (max3903) [inline product_operating_unit/models/product_template.py:24]: @BT-rmartin What do you think of defining the OU on the product category and have a related field on product template?

**Comment by nikul-serpentcs** [comment]
@BT-rmartin If you don't mind, I will work on this PR?

File: product_operating_unit/models/product_template.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    _inherit = 'product.template'
+
+    def _default_operating_unit_ids(self):
+        return [(6, 0,
+                 [self.env['res.users'].operating_unit_default_get(
+                     self.env.uid).id])]
+
+    operating_unit_ids = fields.Many2many(
+        'operating.unit', 'product_operating_unit_rel',
+        string='Operating Unit',
```
Comment (max3903) [inline product_operating_unit/models/product_template.py:22]:
```suggestion
        string='Operating Units',
```

**Comment by max3903** [comment]
@nikul-serpentcs Sounds like you can start your own pull request from this branch...

**Comment by nikul-serpentcs** [comment]
superseded https://github.com/OCA/operating-unit/pull/225

@max3903 @BT-rmartin  can you please close this PR?

**Pull request closed without merging**